src/components/link-preview/link-preview.component.tsx

```diff
@@ -14,6 +13,30 @@ import IconButton from "../icon-button";
 import Icon from "../icon";
 import Placeholder from "./__internal__/placeholder.component";
 
+interface ImageShape {
+  /** The url string to be passed to image src */
+  url: string;
+  /** The string to be passed to image alt */
+  alt?: string;
+}
+
+export interface LinkPreviewProps {
+  /** Used to set the root element to either an anchor link or div container */
+  as?: "a" | "div";
+  /** The description to be displayed */
+  description?: string;
+  /** The config for the image to be displayed */
+  image?: ImageShape;
+  /** Flag to trigger the loading animation */
+  isLoading?: boolean;
+  /** The callback to handle the deleting of a Preview, to hide the close button do not set this prop */
+  onClose?: (url?: string) => void;
+  /** The title to be displayed */
+  title?: string;
+  /** The url string to be displayed and to serve as the link's src */
+  url?: string;
+}
+
 const SCHEME_SEPARATOR = "://";
 
 const LinkPreview = ({
```

Comment:
```suggestion
export const LinkPreview = ({
```

We normally add this when converting to TS. (I think it's because without it Storybook for some reason doesn't show the props table correctly - but I just checked and it seems to here even without this. Not sure what's going on but probably safest to include it.)

Reply:
good spot, looks like I forgot about that this time